### PR TITLE
Added unit tests for several services

### PR DESCRIPTION
**Pull request by anweshan**
Added unit tests to improve test coverage for several services like conversation, text to speech

image

### CODE REVIEW

File: conversation/v1.ts [hunk 1]
```diff
@@ -34,30 +34,12 @@ class ConversationV1 extends GeneratedConversationV1 {
...
-        path: path
-      },
-      defaultOptions: extend(true, this._options, {
-        headers: {
-          accept: 'application/json'
-        }
-      })
-    };
-    return createRequest(parameters, _callback);
+    console.warn(ConversationV1.removedError);
```
Comment (ammardodin) [inline conversation/v1.ts]: Nice job 🔥 @anweshan

File: conversation/v1.ts [hunk 2]
```diff
@@ -221,7 +203,7 @@ class ConversationV1 extends GeneratedConversationV1 {
   }
 
   updateSynonym(params, callback) {
-    if (params && params.new_synonym) {
+    if (params && (params.new_synonym || !params.old_synonym)) {
```
Comment (ammardodin) [inline conversation/v1.ts]: Good catch 👍

File: examples/conversation_tone_analyzer_integration/tone_detection.js [hunk 3]
```diff
@@ -16,26 +16,26 @@
 
 'use strict';
 
-var Promise = require('bluebird');
+const Promise = require('bluebird');
```
Comment (ammardodin) [inline examples/conversation_tone_analyzer_integration/tone_detection.js]: Don't use `const`, use `var` for es5 compatibility.

File: examples/conversation_tone_analyzer_integration/tone_detection.js [hunk 4]
```diff
@@ -152,9 +143,9 @@ function initUser() {
  * @param emotionTone a json object containing the emotion tones in the payload returned by the Tone Analyzer
  */
 function updateEmotionTone(user, emotionTone, maintainHistory) {
-  var maxScore = 0.0;
-  var primaryEmotion = null;
-  var primaryEmotionScore = null;
+  let maxScore = 0.0;
```
Comment (ammardodin) [inline examples/conversation_tone_analyzer_integration/tone_detection.js]: don't use `let`, use `var` for es5 compatibility.

File: .eslintignore [hunk 5]
```diff
@@ -9,5 +9,6 @@ examples/**/node_modules/
 test/resources/auth.js
 # ignore emitted js
 **/*v*.js
+!test/**/*.js
```
Comment (ammardodin) [inline .eslintignore]: Good catch 🔥

File: package.json [hunk 6]
```diff
@@ -65,6 +65,7 @@
     "nyc": "^11.4.0",
     "object.assign": "^4.0.4",
     "prettier": "^1.8.2",
+    "prettier-eslint-cli": "^4.4.2",
```
Comment (germanattanasio) [inline package.json]: no need for this

**Pull request merged**